test/e2e_node/oomkiller_linux_test.go

```diff
@@ -90,13 +93,26 @@ var _ = SIGDescribe("OOMKiller [LinuxOnly]", framework.WithNodeConformance(), fu
 			podSpec: getOOMTargetPod("oomkill-target-pod", "oomkill-multi-target-container",
 				getOOMTargetContainerMultiProcess),
 		})
+
+		testCases = append(testCases, testCase{
+			name:                   "multi process container (single process kill enabled)",
+			oomTargetContainerName: "oomkill-multi-target-container",
+			podSpec: getOOMTargetPod("oomkill-target-pod", "oomkill-multi-target-container",
+				getOOMTargetContainerMultiProcess),
+			enableSingleProcessKill: true,
+			expectPodRunning:        true,
+		})
 	}
 	for _, tc := range testCases {
 		runOomKillerTest(f, tc, 0)
 	}
 })
 
 func runOomKillerTest(f *framework.Framework, testCase testCase, kubeReservedMemory float64) {
+	if kubeReservedMemory > 0 && testCase.enableSingleProcessKill {
+		framework.Failf("only one of kubeReservedMemory and enableSingleProcessKill can be set")
```

Comment:
I'm asking because I'm not familiar with the `kubeReservedMemory` config. Could you document why we validate that only one of the config is set? Is it because we don't want to accidentally mix up the test cases?

Reply:
I have updated the test, making this validation unnecessary.